src/components/CustomerDetails/ManualPassportInput.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, FunctionComponent, useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+import { InputWithLabel } from "../Layout/InputWithLabel";
+import { size } from "../../common/styles";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { DropdownFilterInput } from "../DropdownFilterModal/DropdownFilterInput";
+import { nationalityItems } from "../DropdownFilterModal/nationalityItems";
+import { DropdownItem } from "../DropdownFilterModal/DropdownFilterModal";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center"
+  },
+  inputAndButtonWrapper: {
+    marginTop: size(3),
+    flexDirection: "row",
+    alignItems: "flex-end"
+  },
+  inputWrapper: {
+    flex: 1,
+    marginRight: size(1)
+  }
+});
+
+interface ManualPassportInput {
+  setIdInput: (id: string) => void;
+  submitId: () => void;
+}
+
+export const ManualPassportInput: FunctionComponent<ManualPassportInput> = ({
+  setIdInput,
+  submitId
+}) => {
+  const [selectedCountry, setSelectedCountry] = useState<DropdownItem>();
+  const [passportNum, setPassportNum] = useState<string>();
+
+  const onItemSelection = (item: DropdownItem): void => {
+    setSelectedCountry(item);
+  };
+
+  useEffect(() => {
+    selectedCountry && passportNum
+      ? setIdInput(`${selectedCountry?.id}-${passportNum}`)
+      : setIdInput("");
+  }, [selectedCountry, passportNum, setIdInput]);
+
+  return (
+    <View style={styles.centeredView}>
+      <View style={styles.inputAndButtonWrapper}>
+        <View style={styles.inputWrapper}>
+          <DropdownFilterInput
+            label="Nationality (non-SGP)"
```

Comment:
hmm small nit, not sure if `(non-SGP)` would be clear to the users, wonder if `(tourists)` or `(non-Singaporeans)` or `(visitors)` would be better. Can check with Eida! But doesnt have to be this PR (:

Reply:
Ok let me check with her again. The `non-SGP` is requested by Eida. I feedback the comments to her first

Reply:
they suggested to change to `Nationality (non-Singaporean)`